### ISSUE TITLE
Docker compose build process never ends

### ISSUE THREAD

**Issue by ManuelPuentes** [rairprotocol/rair-dapp#285]
The build process it never ends but it's not trowing errors neither not sure how to fixed this

**Comment by sarora180673** [comment]
Instead of building Docker images locally, application can be started using Docker images available on Docker Hub also using the command
docker-compose -f docker-compose-web.yml up -d

**Comment by sarora180673** [comment]
@ManuelPuentes Yes, you need to copy .env.sample to .env before executing the docker-compose command.
.env.sample already has bare minimum environment variables needed for starting the application
